editor/src/clj/editor/lsp.clj

```diff
@@ -210,8 +210,7 @@
       (let [remaining-responses (dec (requests responses-ch))]
         (cond
           (:error response) (log/warn :message "Language server responded with error" :server server :error (:error response))
-          (nil? (:result response)) (log/info :message "Language server returned no response" :server server)
```

Comment:
There is no point in logging `nil` responses from language servers. With hover responses it even becomes spammy...